finish/src/main/java/io/openliberty/guides/rest/User.java

```diff
@@ -0,0 +1,22 @@
+package io.openliberty.guides.rest;
+
+import java.util.Set;
+
+public class User {
```

Comment:
@NottyCode mentioned that we should not have a sample app that looks like a user registry. Instead, I could change this to be a "Book" repository which I believe is the canonical sample used for JAX-RS anyway